src/test/resources/features/account.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Accounts
+
+  Background:
+    Given I sign in as "owner"
+    And I create an account with :
+      |AccountName    | AccountTest |
+      |Owner    | default |
+    And I capture accountID
+    And In ManageAccountPage select the tab "Projects"
+    And I create project from account manage page with "ProjectAccountTest"
+
+  Scenario: Delete an account
+    When I go to accounts page
+    And  I go to manage account page according to accountID
+    And  In ManageAccountPage select the tab "Settings"
+    And I click on delete account
+    Then I validate that the message "AccountTest was successfully deleted." is displayed
+    And I validate the account with accountID is not present in account list
+    And I validate that account with name "AccountTest" is not in project form
+    And I validate that account with accountID is not present in project settings "MyProject"
```

Comment:
keep in mind 
you test should run in different environment
so now this is failing because it doesn't exist that project "MyProject"

        And I validate that account with accountID is not present in project settings "MyProject" # AccountSteps.iValidateThatAccountWithAccountIDIsNotPresentInProjectSettings(String)
          org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: By.xpath: //a[text()='MyProject']/../following-sibling::div/span[@data-balloon='Project settings'] (tried for 30 second(s) with 500 milliseconds interval)

Reply:
You are right, the problem is that in the precondition I created a project from the account page and when the account is deleted then the projects associated with this account are deleted also.

I will fix this creating the project in the validation, thanks a lot.

Reply:
**NOTA**
The creation of the project would not run in any environment either since in the creation of the project we assume that account1 already exists.